File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split('\n');

let res = [];
const [N, X] = input[0].split(" ").map(Number);
const A = input[1].split(" ").map(Number);

for (let i = 0; i < N; i++) {
  A[i] < X && res.push(`${A[i]}`);
}

console.log(res.join(" "));

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.swift
let cond = readLine()!.split(separator: " ")
let x = Int(cond[1])!
let input: [Int] = readLine()!.split(separator: " ").map{ Int($0)! }
var res: [String] = []

for num in input {
  if num < x {
    res.append("\(num)")
  }
}
print(res.joined(separator: " "))